README.md

```diff
@@ -83,6 +84,7 @@ alternatives, but are high priorities for Formaltiy.
 
 Usage
 -----
+[![formality-js](https://img.shields.io/npm/v/formality-js)](https://www.npmjs.com/package/formality-js)
 
 1. Install
 
```

Comment:
I think there's a one word typo [here](https://github.com/moonad/Formality#portability) as well:
> For example, **out** Formality-to-Haskell compiler was developed

Looks like it might have been meant to be
> For example, **our** Formality-to-Haskell compiler was developed

Reply:
thanks for noticing it xD